core/control-plane/control-plane-transfer/src/main/java/org/eclipse/edc/connector/controlplane/transfer/process/TransferProcessManagerImpl.java

```diff
@@ -202,29 +201,33 @@ private boolean processInitial(TransferProcess process) {
             return true;
         }
 
-        ResourceManifest manifest;
         if (process.getType() == CONSUMER) {
-            var provisioning = dataFlowManager.provision(process, policy);
-            if (provisioning.succeeded()) {
-                var response = provisioning.getContent();
-                if (response.isProvisioning()) {
-                    process.setDataPlaneId(response.getDataPlaneId());
-                    process.transitionProvisioningRequested();
-                } else {
-                    process.setDataPlaneId(null);
-                    process.transitionRequesting();
-                }
-
-                update(process);
-                return true;
-            }
-
             var manifestResult = manifestGenerator.generateConsumerResourceManifest(process, policy);
             if (manifestResult.failed()) {
                 transitionToTerminated(process, format("Resource manifest for process %s cannot be modified to fulfil policy. %s", process.getId(), manifestResult.getFailureMessages()));
                 return true;
             }
-            manifest = manifestResult.getContent();
+            var manifest = manifestResult.getContent();
+
+            if (manifest.empty()) {
```

Comment:
shouldn't this be **not** empty?

Reply:
@ronjaquensel the idea is that the new provisioning flow on data plane gets triggered only if the legacy provisioning in the control-plane does not provision anything.
So in the case there's a provisioning in place (like the s3 or blobstorage) it will continue to work in the same way, otherwise it will call the data-plane to trigger the provisioning there, if there's any provisioner registered.

This condition will be removed once the legacy provisioning will be removed